core/src/main/java/org/apache/accumulo/core/spi/metrics/LoggingMeterRegistryFactory.java

```diff
@@ -49,6 +54,21 @@
  * <pre>
  *     general.custom.metrics.opts.logging.step = 10s
  * </pre>
+ *
+ * To filter out meters that start with any particular prefix, set
+ * {@code general.custom.metrics.opts.meter.id.filters} in the Accumulo configuration.
+ *
+ * <pre>
+ *      general.custom.metrics.opts.meter.id.filters = prefix1,prefix2,prefix3
+ *
+ * </pre>
+ *
+ * To assign a custom delimiter to filters, set
+ * {@code general.custom.metrics.opts.meter.id.delimiter} in the Accumulo configuration.
```

Comment:
Instead of making the delimiter configurable, a different way of doing this, one that is more consistent with how we have set other properties, would be to support something like:

```
general.custom.metrics.loggingfactory.filter.exclude.1=prefix1
general.custom.metrics.loggingfactory.filter.exclude.2=prefix2
general.custom.metrics.loggingfactory.filter..exclude.3=prefix3
```

That way you don't have to worry about a delimiter at all.

Alternatively, make it a single regex filter:

```
general.custom.metrics.loggingfactory.filter.deny.regex=^(prefix1|prefix2|prefix3).*$
```